.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
this should already be gitignored, and as such, eslint should already ignore it - why is this needed?